packages/contract/readme.md

```diff
@@ -0,0 +1,57 @@
+# Tipping bot contract
+
+This contract handles all on chain logic and analytics for the tipping bot.
+
+It does not implement any storage cost logic, meaning that the owner of the contract must provision all storage deposits for the proper functioning of the contract. Storage is mostly required for analytics functionalities.
```

Comment:
Won't users pay when they transfer ?

Reply:
Currently no, they'd pay only for the transfer that they're performing.
It would be possible to post a registration requirement for new users. They'd need to do a storage_deposit call before doing any transfers, however, I don't believe such complications are necessary for the milestone (would require significant logic changes in FE).